nixos/modules/hardware/video/nvidia.nix

```diff
@@ -26,6 +26,8 @@ let
   nvidiaPersistencedEnabled =  cfg.nvidiaPersistenced;
   nvidiaSettings = cfg.nvidiaSettings;
   busIDType = types.strMatching "([[:print:]]+[\:\@][0-9]{1,3}\:[0-9]{1,2}\:[0-9])?";
+
+  ibtSupport = cfg.open || (nvidia.overrideIbtSupport or (lib.versionAtLeast nvidia_x11.version "530"));
```

Comment:
This is not how the `or` operator works:
```
nix-repl> foo = { overrideIbtSupport = null; }

nix-repl> foo.overrideIbtSupport or true
null
```

Reply:
New revision should behave like this:
* if driver has `ibtSupport` set use that, otherwise derive from driver version.
* If the driver doesn't have an `ibtSupport` attribute, default to off for safety.